module #3. REST API Advanced/rest-advanced/build.gradle

```diff
@@ -39,25 +25,17 @@ allprojects {
 
     dependencies {
         implementation 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
Why was this dependency not moved to the rest-app module?

Reply:
My mistake, fixed.